xtc/src/ledger.rs

```diff
@@ -141,14 +142,16 @@ async fn mint(account: Option<Principal>) -> Result<TransactionId, MintError> {
     let account = account.unwrap_or(caller);
     let available = ic.msg_cycles_available();
     let accepted = ic.msg_cycles_accept(available);
+    let fee = compute_fee(accepted).min(accepted);
```

Comment:
what if I only send 1 cycle? The fee calculated here will return 1, right? So someone can cause our wallet to use up a lot of cycles and pay only 1 fee

Reply:
Yeah, but during a mint the user might not have any other cycles that we should withdraw from, do you think we should panic in this case?